lib/vendor/lime/lime.php

```diff
@@ -934,6 +934,10 @@ public function run()
 
       $relative_file = $this->get_relative_file($file);
 
+      if(!is_dir($this->options['test_path'])) {
+        mkdir($this->options['test_path']);
```

Comment:
`Emits an E_WARNING level error if the relevant permissions prevent creating the directory.`

Are we okay with this? I think we are, but just to be sure...

Reply:
I think we are, the path for the (temporary) test output should always be writable.